grails-app/controllers/au/org/ala/specieslist/WebServiceController.groovy

```diff
@@ -34,21 +50,129 @@ class WebServiceController {
 
     def index() { }
 
+    @Operation(
```

Comment:
8 space indentation has crept in - ALA coding std is 4 space indentation.

Reply:
Updated